frontend/packages/kubevirt-plugin/src/selectors/vmi/selectors.ts

```diff
@@ -10,3 +14,9 @@ export const getVMIApiPath = (vmi: VMIKind) =>
   `${VirtualMachineInstanceModel.apiVersion}/namespaces/${getNamespace(vmi)}/${
     VirtualMachineInstanceModel.plural
   }/${getName(vmi)}`;
+
+export const getVMIPVCSourceByDisk = (vmi: VMIKind, diskName: string) =>
+  getVolumePersistentVolumeClaimName(getVMIVolumes(vmi).find((vol) => vol.name === diskName));
+
+export const getVMIBootableDisks = (vmi: VMIKind, defaultValue: V1Disk[] = []): V1Disk[] =>
+  getVMIDisks(vmi, defaultValue).filter((disk) => !Object.keys(disk).includes('serial'));
```

Comment:
why are we ignoring disks with serial?

Reply:
disks with serial are environment variable disks (cm, secret, sa) which aren't bootable

Reply:
Other types of disks can't have serial filled?